### DISCUSSION TITLE
How to specify a value for send_dice method?

### DISCUSSION THREAD

**Discussion by matryosha** [python-telegram-bot/python-telegram-bot#3635] [Q&A] [answered]
As I want to specify exact value for a dice how should I do this? https://github.com/python-telegram-bot/python-telegram-bot/blob/92cb6f3ae8d5c3e49b9019a9348d4408135ffc95/telegram/bot.py#L5315-L5326

I also tried telegram.Dice class as it accepts emoji and **value** but using it with send_dice method throws exception that Dice cannot be serialized.

**Comment by Bibo-Joshi** [comment, marked as answer]
You can't. The bot API doesn't support that. The closest you can get is forwarding a message that contains a die with the desired value.

**Reply by matryosha** [reply]
The bot api definitely [support](https://core.telegram.org/api/dice) it.  As for python-telegram-bot I am not sure but if the lib does not support it then docs for send_dice is misleading because it says otherwise: https://github.com/python-telegram-bot/python-telegram-bot/blob/92cb6f3ae8d5c3e49b9019a9348d4408135ffc95/telegram/bot.py#L5333-L5335

**Reply by matryosha** [reply]
Did I misunderstood the whole thing assuming it can be used for sending values but that's only for receiving a result value after one of the dice sticker has been sent?

**Reply by matryosha** [reply]
> Did I misunderstood the whole thing assuming it can be used for sending values but that's only for receiving a result value after one of the dice sticker has been sent?

I did. It clearly says **random value** in the official api and in python-telegram-bot.
Also I found that there was a question in stackoverflow with the same inquire as mine so it would not be a bad idea to add the remark about it in docs.

**Reply by treasur08** [reply]
> > Did I misunderstood the whole thing assuming it can be used for sending values but that's only for receiving a result value after one of the dice sticker has been sent?
> 
> I did. It clearly says **random value** in the official api and in python-telegram-bot. Also I found that there was a question in stackoverflow with the same inquire as mine so it would not be a bad idea to add the remark about it in docs.

did u fix it

**Reply by matryosha** [reply]
@treasur08 There wasn't really anything to fix as I misunderstood the method in question.

**Reply by realrohitshah-coder** [reply]
Dice
